frontend/src/pages/spotnet/dashboard/Dashboard.jsx

```diff
@@ -192,7 +88,11 @@ const Dashboard = ({ walletId }) => {
         ))}
       </div>
       <div>
-        <button className="btn redeem-btn border-0" onClick={() => closePositionEvent()}>
+        <button
+          className="btn redeem-btn border-0"
+          onClick={() => closePositionMutation.mutate()}
+          disabled={closePositionMutation.isLoading}
```

Comment:
are you sure this works? `isLoading` doesn't exist on closePositionMutation, so it is most likely undefined, please use `isPending` instead